assets/js/index.js

```diff
@@ -42,6 +42,21 @@ async function deleteTodo(taskId) {
   }
 }
 
+async function updateStatus(taskId, isDone) {
+  try {
+    await fetch(`${url}/todos/${taskId}`, {
+      method: "PATCH",
+      body: JSON.stringify({ isDone }),
+      headers: {
+        "Content-Type": "application/json",
+      }
```

Comment:
This part of the code seems to be duplicated. Could you please move this to the global scope and re-use it? Something like `requestDefaultHeaders`.

Reply:
Done.